ocs_ci/framework/conf/default_config.yaml

```diff
@@ -245,6 +245,8 @@ UPGRADE:
 # or ping in ocs infra chat room.
 AUTH:
   test_quay_auth: test_secret
+  pagerduty:
+    escalation_policy: "Default"
```

Comment:
Does it really belong in the AUTH section of the config file?

Reply:
It is a name of escalation policy related to our account (with rules who will get the pager). It is not a token but I think that it could be kept there as it is an account related info and we have a pagerduty section in the config.